code/game/objects/items/stacks/cable.dm

```diff
@@ -168,6 +168,51 @@ var/global/list/datum/stack_recipe/cable_recipes = list ( \
 				getFromPool(/obj/item/stack/cable_coil, C.loc)
 				returnToPool(C)
 
+//I really wish that redundant shit wasn't needed, but snowflake code yeah
+/obj/item/stack/cable_coil/proc/catwalk_place(obj/structure/catwalk/Catwalk, mob/user)
```

Comment:
It would most likely be easier to leave it this way, turf_place has some sanity checks that don't work if you place a cable on a space tile (which is what the new catwalk does). A snowflake proc is better in this regard